index.js

```diff
@@ -0,0 +1,13 @@
+// Лизе нужно написать функцию sayHello(name), которая возвращает приветствие для пользователя. Тем не менее, она влюблена в 
+// пользователя с именем "Mark", и хотела бы поприветствовать его немного иначе.
+
+
+const sayHello = function(name) {
+    if (name === 'Mark') {
+        return ( `Hi, ${name}` );
+    }
+
+    return ( `Hello, ${name}` );
+}
```

Comment:
What if I pass lower cased word 'mark'. Seems that code will not work as expected. Please, add logic to ignore case sensitive issue